src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -644,7 +644,14 @@ DistributedTableSizeOnWorker(WorkerNode *workerNode, Oid relationId,
 	StringInfo tableSizeStringInfo = (StringInfo) linitial(sizeList);
 	char *tableSizeString = tableSizeStringInfo->data;
 
-	*tableSize = SafeStringToUint64(tableSizeString);
+	if (strlen(tableSizeString) > 0)
+	{
+		*tableSize = SafeStringToUint64(tableSizeString);
+	}
+	else
+	{
+		*tableSize = 0;
```

Comment:
could use a comment on when this happens

(I assume it happens because the shard was moved away and dropped just when the citus_total_relation_size started)